recipes/vaex/meta.yaml

```diff
@@ -0,0 +1,103 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "vaex" %}
+{% set version = "1.0.0b2" %}
+{% set sha256 = "25e46cad930a06786de5f4d6a877c91280051f1d52ad9e2727863057f7387932" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    - pyopengl >=3.1.0
+    - numpy >=1.7.2
+    - scipy >=0.1
+    - astropy >=0.3
+    - h5py >=2.2.1
+    - jprops
+    - matplotlib >=1.3.1
+    - numexpr
+    - psutil >=1.2.1
+    - aplus
+    - tornado >4.1
+    - futures
+    - future >=0.15.2
+    - pyyaml
+    - cachetools
+    - progressbar2
+    - attrdict
+    - pandas
+    - pillow
```

Comment:
Please drop all of these that are not strictly needed at build time. Likely just `python` and `setuptools`.